tests/e2e/framework/framework.go

```diff
@@ -58,18 +60,33 @@ type runnable interface {
 	Run() int
 }
 
-// InitGlog sets the logging directory.
+// InitLogging sets the logging directory.
 // Should be called right after flag.Parse().
-func InitGlog() error {
+func InitLogging() error {
```

Comment:
@kyessenov Also please review here

Reply:
I'm not sure this should be done. @geeknoid?

Reply:
So a couple things:

- At the moment, Mixer does none of that. Output go to stdout unless the user redirects it to a specific file.

- Uses of glog in Mixer need to be configured independently using glog options.

- I'm playing right now with providing a replacement for glog which calls into the zap library directly. This would eliminate all classic glog command-line options and features from our builds.

- I'm expecting to extend our logging package to support log rotation explicitly. It's fairly easy to do on top of zap. Adding this functionality will involve introducing new cmd-line flags to allow the user to select a target directory, max file size, num files to keep, etc.

So given all the above, I would probably just go with a simple story for the time being, like Mixer has. The smarts will then be introduced by silent upgrades to the log package.

Reply:
@geeknoid sgtm. I've updated the PR to remove some of the glog-specific gunk.